pkg/solana/logpoller/log_poller.go

```diff
@@ -229,6 +229,8 @@ func (lp *Service) Process(ctx context.Context, programEvent ProgramEvent) (err
 			log.ExpiresAt = &expiresAt
 		}
 
+		lp.lggr.Infof("found matching event %v", log)
```

Comment:
```suggestion
		lp.lggr.Infow("found matching event", "log", log)
```

Reply:
We should always use `w` variants over `f` when using logger so it uses structured logging